lib/tower_cli/models/__init__.py

```diff
@@ -12,9 +12,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+# flake8: noqa
```

Comment:
I tend to avoid module-level exclusions, but it seems to make sense for this case.